R/error-checks.R

```diff
@@ -212,21 +200,31 @@ hasOnlyDistinctValues <- function(values, na.rm = TRUE) {
 
 hasUniqueValues <- hasOnlyDistinctValues
```

Comment:
Is this exported? If yes - I am against it, as the user will see a function that is clearly misnamed. Why do we have to keep this alias? @msevestre what's your oppinion?

Reply:
> Why do we have to keep this alias?

We need to keep the alias as long as all instances of `hasUniqueValues()` have been replaced with `hasOnlyDistinctValues()`.

Otherwise, this will become a breaking change.

Reply:
@pchelle Can you create an issue for that? I have no idea where this is used..

Reply:
Huh, I just checked, and this function is surprisingly not used anywhere else in the ecosystem:
https://github.com/search?q=org%3AOpen-Systems-Pharmacology+hasUniqueValues&type=code

So we don't even need to do aliasing.

Reply:
It's in the Reporting engine?

![image](https://user-images.githubusercontent.com/25903040/153881353-6d72bbc6-c86d-4ce8-bb8c-d36b207a2a9a.png)

Reply:
I have made a PR to replace them:
https://github.com/Open-Systems-Pharmacology/OSPSuite.ReportingEngine/pull/702